runtime/kusama/src/lib.rs

```diff
@@ -1321,7 +1321,7 @@ impl parachains_initializer::Config for Runtime {
 
 impl parachains_disputes::Config for Runtime {
 	type Event = Event;
-	type RewardValidators = ();
+	type RewardValidators = parachains_reward_points::RewardValidatorsWithEraPoints<Runtime>;
```

Comment:
Interesting: This looks like we had no era points previously on Kusama?

Reply:
This is only for disputes.